flashinfer/trace/templates/moe.py

```
@@ -481,6 +482,82 @@ def _trtllm_fp8_block_scale_moe_topk_routing_reference(
+    gemm1_out_size: int = 0,
+    num_hidden_blocks: int = 0,
+    num_intermediate_blocks: int = 0,
+    num_gemm1_out_blocks: int = 0,
+    device: str = "cuda",
+    seed: int = 0,
+):
+    """Build inputs for ``trtllm_fp8_block_scale_moe`` (non-DS routing).
+
+    Sourced from ``tests/moe/test_dpsk_fused_moe_fp8.py`` (the
+    ``_make_inputs_*`` fixture): activations and weights are sampled
+    from ``randn`` (with a ``2.0×`` scale on activations matching the
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)